imports/stylesheets/footer.import.styl

```diff
@@ -6,17 +6,29 @@ footer
   border-top 2px solid $secondary
   background white
   height $footer-height
-  a
-    bg('/images/eha-logo', 280px, 78px, 'svg')
-    display block
-    margin 10px auto
   .eha
     text-align center
+    margin-left 2%
     img
       margin-bottom 15px
+    a
+      bg('/images/eha-logo', 280px, 78px, 'svg')
+      display block
+      margin 10px auto
+  .contact
+    font-weight bold
+    margin-top 40px
+    color darkslategray
+    margin-right 2%
+  .bullet::before
+    padding 20px
+    content "\f111"
+    font-family FontAwesome
```

Comment:
I think this css can be avoided by using the fontawesome html classes: `.fa.fa-circle`